src/golang/cmd/server/handler/v2/dag_get.go

```diff
@@ -55,16 +54,14 @@ func (h *DAGGetHandler) Prepare(r *http.Request) (interface{}, int, error) {
 		return nil, statusCode, err
 	}
 
-	workflowIDStr := chi.URLParam(r, routes.WorkflowIDParam)
-	workflowID, err := uuid.Parse(workflowIDStr)
+	workflowID, err := (parser.WorkflowIDParser{}).Parse(r)
```

Comment:
See comment below about having a generic ID parser so we don't need to initialize unused IDParsers here.